cdxev/validator/validate.py

```diff
@@ -169,3 +172,21 @@ def validate_sbom(
                 )
             )
         return 1
+
+
+def validate_filename(
```

Comment:
funny to add this function, after removing it in #108, where it was already in the [helper.py](https://github.com/Festo-se/cyclonedx-editor-validator/pull/108/files#diff-01abb36cd923cdb942c9c9fde54f042831bcd78bcb76283a340195adbbe6fafeL56) 😆

Reply:
That's called agile 😁 
I'm not taking this a request to change anything, correct?